src/test/java/hudson/remoting/DiagnosedStreamCorruptionExceptionTest.java

```diff
@@ -56,8 +56,9 @@ private void verify(ClassicCommandTransport ct) throws IOException, ClassNotFoun
     @Test(timeout=3000)
     public void blockingStreamShouldNotPreventDiagnosis() throws Exception {
         FastPipedInputStream in = new FastPipedInputStream();
-        FastPipedOutputStream out = new FastPipedOutputStream(in);
-        out.write(payload);
+        try (FastPipedOutputStream out = new FastPipedOutputStream(in)) {
```

Comment:
Shouldn't both the stream opening statements be inside the try with resources?
```suggestion
        try (FastPipedInputStream in = new FastPipedInputStream();
            FastPipedOutputStream out = new FastPipedOutputStream(in)) {
```